packages/firebase_performance/ios/Classes/FirebasePerformancePlugin.h

```diff
@@ -1,4 +1,21 @@
+#import <Firebase/Firebase.h>
 #import <Flutter/Flutter.h>
 
 @interface FLTFirebasePerformancePlugin : NSObject <FlutterPlugin>
++ (void)addMethodHandler:(NSNumber *_Nonnull)handle
```

Comment:
By splitting things into multiple files, you're exposing a lot more class and method surface area here than we were previously. This could become a maintenance burden in the future if people use these APIs and we have to worry about breaking them. I think we should try to make the classes/methods private. Perhaps move `addMethodHandler` to a file named  `FirebasePerformancePlugin+Internal.h`.

It might also make sense to move FirebasePerformancePlugin.h into a Public/ subfolder, if you can get that to work with the build system.

See how Firebase CocoaPods are laid out for reference:

https://github.com/firebase/firebase-ios-sdk/tree/master/Functions/FirebaseFunctions